cmd/config/init/genesis.go

```diff
@@ -73,6 +91,27 @@ func getDefaultGenesisParams(denom string) []PathValue {
 		{"app_state.distribution.params.community_tax", "0.00002"},
 		{"app_state.gov.voting_params.voting_period", "300s"},
 		{"app_state.staking.params.unbonding_time", "3628800s"},
+		{"app_state.bank.denom_metadata", []BankDenomMetadata{
+			{
+				Base: denom,
+				DenomUnits: []BankDenomUnitMetadata{
+					{
+						Aliases:  []string{},
+						Denom:    denom,
+						Exponent: 0,
+					},
+					{
+						Aliases:  []string{},
+						Denom:    displayDenom,
+						Exponent: decimals,
+					},
+				},
+				Description: fmt.Sprintf("Denom metadata for %s (%s)", displayDenom, denom),
+				Display:     displayDenom,
+				Name:        fmt.Sprintf("%s%s", strings.ToUpper(displayDenom[:1]), strings.ToLower(displayDenom[1:])),
```

Comment:
I think the name should be just the display denom. no need to capitalize the first letter